pkg/rpcutil/server.go

```diff
@@ -141,16 +141,21 @@ func (h PreRPCHook[T]) logRateLimit(methodName string, req interface{}) {
 	}
 }
 
-func (h PreRPCHook[T]) checkInitialized(respPointer interface{}) (shouldRet bool) {
-	if !h.initialized.Load() {
-		errInRefelct := reflect.ValueOf(&pb.Error{
-			Code: pb.ErrorCode_MasterNotReady,
-		})
-		reflect.Indirect(reflect.ValueOf(respPointer)).Elem().
-			FieldByName("Err").Set(errInRefelct)
-		return true
+func (h PreRPCHook[T]) checkInitialized(respPointer interface{}) (shouldRet bool, err error) {
+	if h.initialized.Load() {
+		return false, nil
+	}
+
+	respStruct := reflect.ValueOf(respPointer).Elem().Elem()
+	errField := respStruct.FieldByName("Err")
+	if !errField.IsValid() {
+		return true, errors.ErrMasterNotInitialized
```

Comment:
Do we need call `GenWithStackByArgs` errors.ErrMasterNotInitialized with `errors.ErrMasterNotInitialized`